web/src/components/data-fields/IndustryDropdown.tsx

```diff
@@ -55,9 +56,10 @@ export const IndustryDropdown = (props: Props): ReactElement => {
     let homeBasedBusiness: boolean | undefined = undefined;
     let cannabisLicenseType = undefined;
 
-    if (!isHomeBasedBusinessApplicable(industryId)) {
-      homeBasedBusiness = false;
-    }
+    homeBasedBusiness =
+      !isHomeBasedBusinessApplicable(industryId) || nexusLocationInNewJersey(state.profileData)
+        ? false
+        : homeBasedBusiness;
```

Comment:
[dust] Would we consider `homeBasedBusiness =
      isHomeBasedBusinessApplicable(industryId) || !nexusLocationInNewJersey(state.profileData)
        ? homeBasedBusiness : false` for readability?